assets/tests/components/app.test.tsx

```diff
@@ -37,7 +36,9 @@ describe("App", () => {
         <App />
       </SocketProvider>
     )
-    expect(wrapper.exists(".c-modal")).toBeTruthy()
+    expect(
+      result.queryByText(/Your connection to Skate has expired./)
+    ).not.toBeNull()
```

Comment:
I used `.not.toBeNull` here for consistency with other tests. I've previously seen `toBeVisible()` used from [jest-dom](https://github.com/testing-library/jest-dom#tobevisible), which I found helpful for brevity when first learning react testing, but I don't feel strongly enough about it to break the existing testing conventions here!

Reply:
I actually wasn't aware of `toBeVisible`, that actually looks better in at least some respects (its assertions cover more cases that might lead to the element not showing up on the page, and the naming is more clear about what we're looking to test). Let's use it going forward!